### ISSUE TITLE
Disabling "start on boot" does not do work

### ISSUE THREAD

**Issue by ElectroBytezLV** [lacymorrow/crossover#296]
Disabling "start on boot" in settings does not get rid of start on windows boot. Have to manually disable it in "startup apps" on windows settings for it to work.

**Comment by lacymorrow** [comment]
Thanks for the report, I'm looking into this.

**Comment by KeOsdenLM** [comment]
Yes, the same thing happens to me too, it doesn't make sense a checkbox to disable the app startup at windows startup, if it doesn't work. I hope you solve it soon, because you have done a great job with this app.
Regards.

**Comment by lacymorrow** [comment]
Yeah I'm sorry about that, I use the features I add (usually) but they don't get tested always on every OS, I'll get this fixed soon. Cheers

**Comment by lacymorrow** [comment]
@openhands-agent

**Comment by lacymorrow** [comment]
📋 Tracked in Paperclip as **LAC-1349**. This issue has been added to our internal project management system and will be reviewed by the engineering team.

**Comment by lacymorrow** [comment]
📋 Tracked in Paperclip as **LAC-1779**. This issue has been added to our internal project management system and will be reviewed by the engineering team.

**Comment by lacymorrow** [comment]
Tracked in Paperclip as **LAC-1349**. This issue has been added to our internal project management system and will be reviewed by the engineering team.

**Comment by lacymorrow** [comment]
This is from late 2023 and the startup behavior has been reworked since then. If autostart still misbehaves on the current version, open a fresh issue with your OS version and we'll dig in.
